paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -227,6 +244,41 @@ public class PaparazziPlugin @Inject constructor(
 
       recordTaskProvider.configure { it.dependsOn(testTaskProvider) }
       verifyTaskProvider.configure { it.dependsOn(testTaskProvider) }
+      addFailedSnapshotsTaskProvider.configure {
+        it.inputs.dir(buildDirectory.dir("paparazzi/failures"))
+          .withPropertyName("paparazzi.snapshot.failures")
+          .withPathSensitivity(PathSensitivity.RELATIVE)
+        it.inputs.dir(testReportOutputLocation)
+          .withPropertyName("paparazzi.html.report.dir")
+          .withPathSensitivity(PathSensitivity.RELATIVE)
+
+        it.mustRunAfter(testTaskProvider)
+
+        it.doLast {
+          val reportRootDir = testReportOutputLocation.get().asFile
```

Comment:
is there a way to customize junit reports outside of the plugin?  how does the android studio team do it?